hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadataUtil.java

```diff
@@ -378,7 +378,8 @@ public static boolean metadataPartitionExists(StoragePath basePath, HoodieEngine
   public static Map<String, HoodieData<HoodieRecord>> convertMetadataToRecords(HoodieEngineContext context, HoodieConfig hoodieConfig, HoodieCommitMetadata commitMetadata,
                                                                                String instantTime, HoodieTableMetaClient dataMetaClient, HoodieTableMetadata tableMetadata,
                                                                                HoodieMetadataConfig metadataConfig, List<MetadataPartitionType> enabledPartitionTypes, String bloomFilterType,
-                                                                               int bloomIndexParallelism, Integer writesFileIdEncoding, EngineType engineType) {
+                                                                               int bloomIndexParallelism, Integer writesFileIdEncoding, EngineType engineType,
```

Comment:
unrelated to the patch - can we keep `writesFileIdEncoding` as primitive `int` instead of `Integer`?